### DISCUSSION TITLE
App groups for easier permissions setting at scale

### DISCUSSION THREAD

**Discussion by danielrosehill** [Budibase/budibase#14058] [Ideas]
Feature suggestion (sorry if it exists on enterprise, I'm a solo shop on the basic plan!):

The ability to organise apps into folders. 

This would make it easier to manage resources by organisation.

It would also be very useful when configuring access rules at scale. 

E.g. group all apps for the finance team into a 'Finance' folder and give permissions on that that recurse into the individual apps (naturally some orgs may want to go more granular so having both options would be helpful).

**Comment by ConorWebb96** [comment]
Hey @danielrosehill,

I think this may fall under [user groups](https://docs.budibase.com/docs/user-groups), this allows you to give assign users to different groups and then use that group to access specific apps. You are also right this is a enterprise tier feature.
